{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Differential Expressed Genes (DEG) and Antimicrobial Resistance Genes Analysis\n",
    "\n",
    "This workflow page is prepared as supplementary code for DEG and Antimicrobial Resistance analysis\n",
    "\n",
    "## Tutorial sections\n",
    "This tutorial comprises the following sections:   \n",
    " 1. [DEG Analysis](2_DEG_analysis.ipynb)\n",
    " 2. [Machine Learning](3_Machine_learning.ipynb)\n",
    " 3. [Antimicrobial Resistance Genes](4_AMR_Genes.ipynb)\n",
    " 4. [Protein-Protein Interaction](5_PPI_analysis.ipynb) \n",
    " 5. [Functional Enrichment Analysis](6_Enrichment_Analysis.ipynb) \n",
    "   "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "id": "bXnRjRgZw9sy"
   },
   "source": [
    "# Introduction\n",
    "In this research, you can use transcriptomic dataset from https://www.ncbi.nlm.nih.gov/geo/."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "id": "1u3-aTpS3Dpq"
   },
   "source": [
    "# General Workflow of Analysis\n",
    "\n",
    "<img src=\"img/1_General_Workflow_analysis.png\"/>\n",
    "\n",
    "<center> General Workflow of DEG and Antimicrobial Resistance Biomarker </center>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "colab": {
   "provenance": []
  },
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
